src/vs/workbench/services/backup/common/backup.ts

```diff
@@ -0,0 +1,88 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Isn't `vs/platform` meant for code that is shared between main and renderer processes? The main service (now `IBackupMainService`) is exclusively used by the main process and the these interfaces are exclusive to the renderer process. Currently `IBackupWorkspacesFormat` is the only thing should be shared and that lives in `vs/platform/backup/common/backup.ts`